pkg/render/policyrecommendation.go

```diff
@@ -197,28 +205,8 @@ func (pr *policyRecommendationComponent) clusterRole() client.Object {
 	}
 }
 
-func (pr *policyRecommendationComponent) clusterRoleBinding() client.Object {
-	return &rbacv1.ClusterRoleBinding{
-		TypeMeta: metav1.TypeMeta{
-			APIVersion: "rbac.authorization.k8s.io/v1",
-			Kind:       "ClusterRoleBinding",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: PolicyRecommendationName,
-		},
-		RoleRef: rbacv1.RoleRef{
-			APIGroup: "rbac.authorization.k8s.io",
-			Kind:     "ClusterRole",
-			Name:     PolicyRecommendationName,
-		},
-		Subjects: []rbacv1.Subject{
-			{
-				Kind:      "ServiceAccount",
-				Name:      PolicyRecommendationName,
-				Namespace: pr.cfg.Namespace,
-			},
-		},
-	}
+func (pr *policyRecommendationComponent) clusterRoleBinding(namespaces []string) client.Object {
```

Comment:
I think this is only used in this file, so we can get rid of this function and just call `rcomponents.ClusterRoleBinding` directly. 

The only reason we have a helper for the manager one is because it's also called from the guardian render code, IIRC